packages/renderer/src/lib/image/ImageActions.svelte

```diff
@@ -39,33 +38,24 @@ let {
 const imageUtils = new ImageUtils();
 
 let contributions: Menu[] = $state([]);
-let globalContext: ContextUI | undefined = $state();
-let contextsUnsubscribe: Unsubscriber;
-let groupingContributions = $state(false);
+
+let groupingContributions = $derived(groupContributions && !dropdownMenu && contributions.length > 1);
+let globalContext: ContextUI | undefined = $derived.by(() => {
```

Comment:
it cannot be undefined from what I read?

```suggestion
let globalContext: ContextUI = $derived.by(() => {
```